design_practice/src/assets/stylesheets/base/variables.sass

```diff
@@ -0,0 +1,69 @@
+$sm: 576px
+$md: 768px
+$lg: 992px
+$xl: 1200px
+    
+//----- fonts sizes and line-heights -----------------------------------------------------------------------------------
+$font-size-xxl:                   72px
+$font-size-xl:                    30px
+$font-size-lg:                    20px
+$font-size-base:                  16px
+$font-size-sm:                    14px
+$font-size-xs:                    12px
+$font-size-xxs:                   10px
+
+$line-height-base:                1.25
+//----- spacings (margins/paddings) ------------------------------------------------------------------------------------
+$m-base:                          20px
+$m-xxs:                           $m-base / 5
+$m-xs:                            $m-base / 4
+$m-sm:                            $m-base / 2
+$m-md:                            $m-base / 4 * 3
+$m-lg:                            $m-base * 1.5
+$m-xl:                            $m-base * 2
+$m-xxl:                           $m-base * 2.5
+
+//----- border radiuses ------------------------------------------------------------------------------------------------
+$border-radius:                   3px
+
+//----- breakpoints ----------------------------------------------------------------------------------------------------
+$grid-breakpoints:                (xs: 0, sm: 576px, md: 768px, lg: 992px, xl: 1200px)
+$container:                       map-get($grid-breakpoints, xl)
+
+//----- colors ---------------------------------------------------------------------------------------------------------
+$primary-gradient-1:              #63c431
+$primary-gradient-2:              #5fc22d
+
+$accent-gradient-1:               #8b32a8
+$accent-gradient-2:               #9342ad
+
+$bar-gradient-1:                  #713fcb
+$bar-gradient-2:                  #8d5fde
+
+$black:                           #04080c
+
+$dark-accent-1:                   #5b32a4
+$dark-accent-2:                   #492789
+
+$gray-20:                         hsl(36, 5%, 20%)
+$gray-30:                         hsl(36, 5%, 30%)
+$gray-40:                         hsl(36, 5%, 40%)
+$gray-50:                         hsl(36, 5%, 50%)
+$gray-75:                         hsl(36, 5%, 75%)
+$gray-90:                         hsl(36, 5%, 90%)
+$gray-98:                         hsl(36, 5%, 98%)
```

Comment:
I used to do it as well :) But if the colors change, the variable names are no longer valid - hence `gray-light`, `lighter` etc. A good example is also `brand-primary` - it does not matter whether it's blue or green